audb/core/info.py

```diff
@@ -250,33 +260,41 @@ def header(
         name: str,
         *,
         version: str = None,
+        load_tables: bool = True,
         cache_root: str = None,
 ) -> audformat.Database:
     r"""Load header of database.
 
     Args:
         name: name of database
         version: version of database
+        load_tables: if ``True``
+            it will download misc tables
+            used as labels in a scheme
```

Comment:
```suggestion
            download misc tables
            used by schemes
```

Reply:
I changed it to

```
            downloads misc tables
            used by schemes
```

and applied the same change in the docstring of `audb.info.schemes()`

Reply:
Sorry, I forgot to push the changes. I will open another pull request fixing this.

Reply:
My bad, I only checked the images in the description.